### DISCUSSION TITLE
[DOC]: migration from conductor 2.x

### DISCUSSION THREAD

**Discussion by hebrd** [Netflix/conductor#3042] [Q&A]
## What are you missing in the docs
Is conductor 3.x compitable with 2.x?
any guide for existing user of 2.x to upgrade to 3.x?

**Comment by aravindanr** [comment]
We don't have a migration guide, but the changes in 3.x are captured in https://github.com/Netflix/conductor/blob/main/CHANGELOG.md. If you run into issues during migration, you can post here.

**Reply by hebrd** [reply]
I just upgraded conductor ui to 3.x and kept server unchanged, It works well so far, the new version of ui is much more organised and convenient.
